test/go.mod

```diff
@@ -4,10 +4,12 @@ go 1.13
 
 require (
 	github.com/blang/semver v3.5.1+incompatible
+	github.com/docker/docker v1.4.2-0.20190916154449-92cc603036dd
 	github.com/emicklei/go-restful v2.11.1+incompatible // indirect
 	github.com/go-openapi/spec v0.19.4 // indirect
+	github.com/google/uuid v1.1.1
 	github.com/imdario/mergo v0.3.8 // indirect
-	github.com/mesosphere/kubeaddons v0.7.2
+	github.com/mesosphere/kubeaddons master
```

Comment:
Can someone cut a kubeaddons release so we don't need to pin master here?

Reply:
This is done, please rebase.

Reply:
Thanks!